lib/openhab/core/items/semantics/semantic_tag.rb

```diff
@@ -93,6 +93,11 @@ def inspect
               "synonyms=#{synonyms.to_a.inspect}>"
           end
 
+          # @return [String]
```

Comment:
alias_method?

Reply:
That's what I tried at first, but it complained that #name doesn't exist, "undefined method" or something. Maybe it's because SemanticTag is an interface?